sogou_tr_async/sogou_tr_async.py

```diff
@@ -97,9 +97,9 @@ def with_attrs(fct: Callable) -> Callable:
 
     if fuzzy:
         if from_lang not in SOGOUTR_CODES:
-            from_lang = process.extractOne(from_lang, SOGOUTR_CODES, scorer=fuzz.UWRatio)[0]  # NOQA
+            from_lang = process.extractOne(from_lang, SOGOUTR_CODES)[0]  # NOQA
         if to_lang not in SOGOUTR_CODES:
-            to_lang = process.extractOne(to_lang, SOGOUTR_CODES, scorer=fuzz.UWRatio)[0]  # NOQA
```

Comment:
Rapidfuzz's WRatio default does not throw away non ascii -> equivalent to UWRatio in fuzzywuzzy